src/server/cluster/cluster_config.h

```diff
@@ -61,7 +64,9 @@ class ClusterConfig {
   // Returns true if `new_config` is valid and internal state was changed. Returns false and changes
   // nothing otherwise.
   bool SetConfig(const ClusterShards& new_config);
```

Comment:
move to private?

Reply:
Generally, I'd agree. I just tried to move it, but it would then require testing solely by passing `json`, which is more cumbersome. Also, I _guess_ it's a nicer API to support both, although I agree that if it's not in use there's no point in keeping it.
Again, I'd love to keep it if only to make testing this class easier, OK?

Reply:
sure